### PR TITLE
11374 Filings UI: get account id from CURRENT_ACCOUNT

### PR DESCRIPTION
**Pull request by lewischenstudio**
*Issue #:* [/bcgov/entity#11374](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/11374)

*Description of changes:*
- removed `accountId` in file: fetch-config.ts 
- updated `accountId` in file: navigate.ts
- updated app version in file: package.json
- updated many unit test files to accommodate `CURRENT_ACCOUNT`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: tests/unit/fetch-config.spec.ts [hunk 1]
```diff
@@ -43,7 +43,6 @@ xdescribe('Fetch Config', () => {
     await fetchConfig()
 
     // verify data
-    expect(sessionStorage.getItem('ACCOUNT_ID')).toBe('2288')
```
Comment (severinbeauvais) [inline tests/unit/fetch-config.spec.ts:46]: Somewhere, we should have a test to verify that the account id is correctly parsed and used, ie:
- in BreadcrumbResources.ts
- in navigate.ts

Maybe app.spec.ts would be a good place? I'd be OK with a new describe section in there just to keep it separate from everything else.

**Review by severinbeauvais — COMMENTED** [review]
Looks really good.

Thanks for the other, small cleanup!

I'm hoping for another unit test but otherwise I see no needed code changes.

**Review by lambert-alex — APPROVED** [review]

File: tests/unit/fetch-config.spec.ts (hunk 1 above)
Comment (severinbeauvais) [reply tests/unit/fetch-config.spec.ts:46]: I'm hoping for some tests that verify that both places this value is used, it's properly parsed from the current account object and they build the correct output string (for breadcrumb or navigate).

Comment (severinbeauvais) [reply tests/unit/fetch-config.spec.ts:46]: Ah, OK, I see now. Looks good.

**Review by vysakh-menon-aot — APPROVED** [review]

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**